akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -187,9 +190,15 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
    */
   def close(): Unit = {
     _closed = true
```

Comment:
I could reproduce the test failure by adding a strategic sleep in the eventBusListener actor